### ISSUE TITLE
Ubuntu 21.04: The configured user limit (128) on the number of inotify instances has been reached, or the per-process limit on the number of open file descriptors has been reached.

### ISSUE THREAD

**Issue by chronossc** [riok/Kreya#34]
**Describe the bug**
Somehow Kreya give me the error `System.IO.IOException: The configured user limit (128) on the number of inotify instances has been reached, or the per-process limit on the number of open file descriptors has been reached.`

My limits:
```
$ ulimit -Hn; ulimit -Sn       
1048576
1024

$ cat /proc/sys/fs/file-max    
9223372036854775807

$ cat /proc/sys/fs/inotify/max_user_watches
1524288
```

**To Reproduce**
1. Install kreya via Snap
3. Run `kreya` and create a project
3. See error
4. Run `kreya` again
5. Open created project
6. See error

**Expected behavior**
I expect it to open without errors.

**Environment (if possible, copy the information from the error dialog or the About menu):**
 - OS:
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.10
Release:        20.10
Codename:       groovy

$ uname -a
Linux ahsoka 5.8.0-50-generic #56-Ubuntu SMP Mon Apr 12 17:18:36 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```
 - Kreya Version: 1.4.0

**Additional context**
Full log:
```
{
  "kreyaVersion": "1.4.0",
  "platform": "Linux x86_64",
  "userAgent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Safari/605.1.15"
}

System.IO.IOException: The configured user limit (128) on the number of inotify instances has been reached, or the per-process limit on the number of open file descriptors has been reached.
   at System.IO.FileSystemWatcher.StartRaisingEvents()
   at System.IO.FileSystemWatcher.StartRaisingEventsIfNotDisposed()
   at System.IO.FileSystemWatcher.set_EnableRaisingEvents(Boolean value)
   at Kreya.Data.FileSystem.FileWatcher.ProjectFileWatcherService.Watch(IProject proj)
   at Kreya.Core.Projects.ProjectAccessor.InitActiveProject(Project proj)
   at Kreya.Core.Projects.ProjectService.LoadProject(String path)
   at Kreya.UI.Projects.ProjectUiService.Open(String path)
   at SpiderEye.Bridge.ApiMethod.InvokeAsync(Object instance, Object parameter)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.ResolveCall(String id, String parameters)
```

**Comment by CommonGuy** [comment]
Thanks for the detailed bug report. We will look into it

**Comment by CommonGuy** [comment]
We just released Kreya 1.5.0-beta.1 which includes this bugfix: https://kreya.app/release-notes/beta. You can switch to the Beta release channel by executing `snap install kreya --beta`.

**Comment by CommonGuy** [comment]
Kreya v1.5 was just released, which includes a fix for this bug.
